platform/thunderbird/manifest.json

```diff
@@ -8,7 +8,7 @@
     },
     "browser_specific_settings": {
         "gecko": {
-            "id": "linux-entra-sso@example.com",
+            "id": "@linux-entra-sso.tb.siemens",
```

Comment:
Thanks @fmoessbauer for the ping. IIUC we don't actually need a real email address anymore for this and the ID format is more desirable?

BTW, I'd have no problem also simply using `opensource@siemens.com` for the email and we just route any emails/requests if needed. But I'm not sure if you're comfortable with that or what @bufferoverflow 's opinion is here.

Reply:
Thanks @fmoessbauer, makes sense to me. So if you need any other contact somewhere feel free to use the global opensource one I'd say. LGTM here then :)